deeppavlov/core/agent_v2/agent.py

```diff
@@ -35,7 +35,13 @@ def __call__(self, utterances: Sequence[str], user_telegram_ids: Sequence[Hashab
 
         skill_names, utterances, confidences = self.skill_manager(state)
 
-        self.state_manager.add_bot_utterances(me_dialogs, utterances, [datetime.utcnow()] * len(me_dialogs),
-                                              skill_names, confidences)
+        bot_utterances = self.state_manager.add_bot_utterances(me_dialogs, utterances,
```

Comment:
`StateManager.add_bot_utterances()` and `StateManager.get_utterances()` should be written in the same style (method name and I think it shouldn't pass dialogs, only texts).

Reply:
`bot_utterances` should be named `me_bot_utterances`